src/components/Chip/components/Badge/Badge.tsx

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import { generateTestDataId } from 'utils/helpers';
 import { TestId } from 'utils/types';
 
-import { Props } from '../../Chip.types';
+import { ChipProps } from '../../Chip.types';
 import { badgeStyle } from './Badge.style';
 
```

Comment:
Isn't this type already exported from somewhere else in ictinus? Is this necessary?